jib-core/src/main/java/com/google/cloud/tools/jib/builder/steps/BuildImageStep.java

```diff
@@ -160,6 +161,8 @@ public DescriptorDigest getDiffId() throws LayerPropertyNotFoundException {
       }
 
       // Add built layers/configuration
+      // Note: ProjectInfo.VERSION may be null in tests
+      String version = ProjectInfo.VERSION == null ? "" : (":" + ProjectInfo.VERSION);
```

Comment:
When null, how about ":unknown-version" or such? That way, we can distinguish between older versions and failing to get version info in newer versions.

Reply:
Good point